pkg/services/ngalert/schedule/schedule_test.go

```diff
@@ -106,12 +106,11 @@ func TestWarmStateCache(t *testing.T) {
 	}
 
 	schedCfg := schedule.SchedulerCfg{
-		Cfg:           cfg,
-		C:             clock.NewMock(),
-		Logger:        log.New("ngalert cache warming test"),
-		RuleStore:     dbstore,
-		InstanceStore: dbstore,
-		Metrics:       testMetrics.GetSchedulerMetrics(),
+		Cfg:       cfg,
+		C:         clock.NewMock(),
+		Logger:    log.New("ngalert cache warming test"),
+		RuleStore: dbstore,
+		Metrics:   testMetrics.GetSchedulerMetrics(),
```

Comment:
`InstanceStore` was dead. Scheduler does not talk to instancestore directly at all. Store is now completely encapsulated by state manager.

Reply:
strange, I though I deleted it in my refactoring... perhaps rebase brought it back

Reply:
Nope, still there in main. Either way, it'll be gone for good now!